### ISSUE TITLE
Not Compatible with Gatsby 5

### ISSUE THREAD

**Issue by rosswhitehouse** [benjaminhoffman/gatsby-plugin-segment-js#51]
Hey!

When I `gatsby develop` locally I get this warning:

```
warn Plugin gatsby-plugin-segment-js is not compatible with your gatsby version 5.0.0 - It requires gatsby@^2 || ^3 || ^4
```

It seems to be working OK for me but I use it just to add the script, so for me it's safe to just to ignore it. I suppose there'd need to be more tests done to check more complicated functionality before declaring that it's fully working with Gatsby 5.

Thank you for your time 🙇

**Comment by brandon-hedrick** [comment]
+1

**Comment by thannes** [comment]
+1

**Comment by newhouse** [comment]
OK OK... 😄

**Comment by newhouse** [comment]
This is now fixed in `3.7.2`. Please confirm @brandon-hedrick @rosswhitehouse @thannes
